mango-viz/bdgenomics/mango/pileup/utils.py

```diff
@@ -1,74 +1,50 @@
 import json
+def parse_bed_dataframe(dataframe):
```

Comment:
Looks good! Can you add unit tests for this?

Reply:
where should I add these unit tests? also, is there an example of one?

Reply:
Tests for mango-viz are at https://github.com/bigdatagenomics/mango/tree/master/mango-viz/bdgenomics/mango/pileup/test. You can add a test-utils.py file with tests

Reply:
remove this file I think it is duplicated.